x-pack/plugins/security_solution/server/lib/telemetry/receiver.ts

```diff
@@ -230,6 +316,9 @@ export class TelemetryReceiver {
       throw Error('exception list client is unavailable: cannot retrieve trusted applications');
     }
 
+    // Ensure list is created if it does not exist
+    await this.exceptionListClient.createTrustedAppsList();
```

Comment:
NOTE: This looks like it was missing within `fetchTrustedApplications` so I added it. The e2e tests also prefer to have this added.